packit/copr_helper.py

```diff
@@ -230,10 +230,19 @@ def create_or_update_copr_project(
                 exist_ok=True,
             )
         except (CoprException, CoprRequestException) as ex:
-            error = (
-                f"Cannot create a new Copr project "
-                f"(owner={owner} project={project} chroots={chroots}): {ex}"
-            )
+            response_status_code = ex.result.__response__.status_code
+            if response_status_code == 500:
+                error = (
+                    "Packit received an HTTP 500 Internal Server Error from Copr Service. "
+                    "Check the Copr status page: https://copr.fedorainfracloud.org/status/stats/, "
+                    "or ask for help in Red Hat internal Copr slack channel #copr."
+                )
+            else:
+                error = (
+                    f"Cannot create a new Copr project "
+                    f"(owner={owner} project={project} chroots={chroots}): {ex}. "
+                    f"Copr http response status code is {response_status_code}."
```

Comment:
```suggestion
                    f"Copr HTTP response status code is {response_status_code}."
```

As above, this could include also textual `reason`, not just the status code.